tests/custom-package-edit-test.js

```diff
@@ -93,7 +93,7 @@ describeApplication('CustomPackageEdit', () => {
         });
 
         it('displays new coverage dates', () => {
-          expect(PackageShowPage.customCoverage).to.equal('12/16/2018 - 12/18/2018');
+          expect(PackageShowPage.customCoverage).to.equal('12/16/2018 - 12/18/2018');
```

Comment:
I'm not exactly sure what happened with white space here, but copy/pasting got the tests to pass.